src/Api/CustomSuggestionsApi.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace LupaSearch\Api;
+
+use LupaSearch\LupaClient;
+
+use function json_encode;
+
+class CustomSuggestionsApi
+{
+    private $client;
+
+    public function __construct(LupaClient $client)
+    {
+        $this->client = $client;
+    }
+
+    public function getCustomSuggestions(string $indexId): array
+    {
+        return $this->client->send(
+            LupaClient::METHOD_GET,
+            LupaClient::API_BASE_PATH . "/indices/$indexId/customSuggestions",
+            true
+        );
+    }
+
+    public function createCustomSuggestions(
+        string $indexId,
```

Comment:
You write that it would support PHP ^7.2, but here it will throw an error due to `string` typehint. It appeared on 7.4. I would fix the readme.

Reply:
Scalar type declarations are available since PHP 7.0, you probably have in mind typed class properties?